SPECS/libtar/libtar.spec

```diff
@@ -59,20 +66,30 @@ chmod +x %{buildroot}/%{_libdir}/libtar.so.*
 %{_libdir}/libtar.la
```

Comment:
Nit: While we're making changes to this spec, would you mind removing the libtool archive file?